templates/wheretofindme/follow_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block "main" %}
+<h1 class="display-4">Where to find everyone else</h1>
+<div>
+  {% for follow in object_list %}
+    <div class="card bg-light shadow-sm mb-3">
```

Comment:
One thing that might be cool here is presenting a pivot table where the rows are followed users, and the columns are drawn from their identities (so you get a column of facebook links, eg.), with CSV export on the side. This would make the who-you-follow page a good one-stop shop for re-connecting with people when a company goes bang.

Reply:
My only worry is the "where to find me" can be _long_. Maybe cap it at 3 entries?

Reply:
What I'm more concerned about is inconsistent labelling. In principle, `twitter, `Twitter`, and `twitter:` should all be categorized together. Facebook is even worse (`facebook`, `Facebook`, `FaceBook`, `face book`, and all the other variations are all more or less equally likely inputs). That'll make it hard to put "the same service" in the same column. There's also recurrences, subcategories (`mastodon (person)`, `mastodon (lewd)`, etc), and other conventions that don't exist on the service yet.

Making it very very wide is obviously not great; making it extra wide because of how users label stuff is maybe worse?

Just showing the first three identities, without trying to categorize them, might be a good compromise though.